Advanced Featuretools RUL.ipynb

```diff
@@ -697,319 +832,303 @@
-       "2                                                   2.740721                                       \n",
-       "3                                                   1.144065                                       \n",
-       "4                                                   4.094572                                       \n",
-       "5                                                   4.327066                                       \n",
+       "           CID_CE(recordings.settings_clusters.CID_CE(recordings.operational_setting_1, normalize=False), normalize=False)  \\\n",
+       "engine_no                                                                                                                    \n",
+       "1                                                   0.008253                                                                 \n",
+       "1                                                   0.011615                                                                 \n",
+       "2                                                   0.009820                                                                 \n",
+       "2                                                   0.011904                                                                 \n",
+       "3                                                   0.015820                                                                 \n",
        "\n",
-       "           RUL  \n",
-       "engine_no       \n",
-       "1           66  \n",
-       "2           70  \n",
-       "3          294  \n",
-       "4           56  \n",
-       "5            7  \n",
+       "           remaining_useful_life  \n",
+       "engine_no                         \n",
+       "1                            210  \n",
+       "1                            130  \n",
+       "2                            198  \n",
+       "2                            158  \n",
+       "3                            136  \n",
        "\n",
-       "[5 rows x 303 columns]"
+       "[5 rows x 305 columns]"
       ]
      },
-     "execution_count": 5,
+     "execution_count": 7,
      "metadata": {},
      "output_type": "execute_result"
     }
    ],
    "source": [
-    "from featuretools.primitives import make_agg_primitive\n",
-    "import featuretools.variable_types as vtypes\n",
-    "\n",
-    "from tsfresh.feature_extraction.feature_calculators import (number_peaks, mean_abs_change, \n",
-    "                                                            cid_ce, last_location_of_maximum, length)\n",
-    "\n",
-    "\n",
-    "Complexity = make_agg_primitive(lambda x: cid_ce(x, False),\n",
-    "                              input_types=[vtypes.Numeric],\n",
-    "                              return_type=vtypes.Numeric,\n",
-    "                              name=\"complexity\")\n",
+    "from featuretools_tsfresh_primitives import CidCe\n",
```

Comment:
After installing tsfresh primitives, using this command.

```
python -m pip install featuretools[tsfresh]
```

I wasn't able to import the tsfresh primitives from the featuretools package.

```
from featuretools.tsfresh import CidCe
```

Reply:
updated notebooks to import tsfresh primitives from featuretools